### PR TITLE
Changelog for 10.19.0

### PR DESCRIPTION
**Pull request by nakulkar-msft**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
 
 # Change Log
 
+
+## Version 10.19.0
+
+### New Features
+
+1. Support for new Cold Tier feature for Azure Blobs
+2. Support preserving a trailing dot ('.') in names of files and directories in Azure Files
```
Comment (gapra-msft) [inline ChangeLog.md:10]: Should we mention here that the default behavior is different?

File: ChangeLog.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+3. Alternate modes to preserve hash for hash-based sync ([#2214](https://github.com/Azure/azure-storage-azcopy/issues/2214))
+4. Support 'force-if-readonly' flag for Sync.
+5. Preserve posix properties while uploading or downloading from HNS enabled accounts
+### Bug Fixes
+
+1. Fix situation where large-files would hang infinitely with low value for 'cap-mbps'
```
[the remark this answers is not in the document]
Comment (gapra-msft) [reply ChangeLog.md:16]: This issue was also fixed https://github.com/Azure/azure-storage-azcopy/issues/1790

**Review by gapra-msft — COMMENTED** [review]
see comments

File: ChangeLog.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
 
 # Change Log
 
+
+## Version 10.19.0
+
+### New Features
+
+1. Support for new Cold Tier feature for Azure Blobs
```
Comment (adreed-msft) [inline ChangeLog.md:9]: We should mention `--block-blob-tier=Cold` here.

File: ChangeLog.md [hunk 4]
```diff
@@ -1,6 +1,23 @@
...
+2. Support preserving a trailing dot ('.') in names of files and directories in Azure Files
+3. Alternate modes to preserve hash for hash-based sync ([#2214](https://github.com/Azure/azure-storage-azcopy/issues/2214))
```
Comment (adreed-msft) [inline ChangeLog.md:11]: Flag name

File: ChangeLog.md [hunk 5]
```diff
@@ -1,6 +1,23 @@
...
+4. Support 'force-if-readonly' flag for Sync.
+5. Preserve posix properties while uploading or downloading from HNS enabled accounts
```
Comment (adreed-msft) [inline ChangeLog.md:13]: ditto

**Review by gapra-msft — APPROVED** [review]

**Review by tasherif-msft — APPROVED** [review]

**Pull request merged**